_emails/newsletter-2021-03.html

```diff
@@ -0,0 +1,148 @@
+{% include email/markdown.html content=text %}
+
+{% capture text %}
+- [Financial-Times/origami-build-service](https://api.github.com/repos/Financial-Times/origami-build-service)
+   - Deprecated the use of `modules` for anything that's not in the allow-list.
+   - Error message "check your typos" updated to "check for typos".
+   - Added autoprefixer for css and using the same browser config as origami-build-tools.
+   - Only allows @financial-times namespaced components to be requested in v3.
+   - Renamed the modules and module query parameters to components and component in v3.
+   - Implemented the /v3/demo and /v3/demo/html endpoints.
+
+- [Financial-Times/o-layout](https://api.github.com/repos/Financial-Times/o-layout)
+   - Add master brand support for default, bleed, documentation layouts.
+   - Support `figure` in the `o-layout-typography` wrapper.
+
+- [Financial-Times/origami-build-tools](https://api.github.com/repos/Financial-Times/origami-build-tools)
+
+   - Add validation for package.json.browser field
+   - Add validation for package.json.name field
+   - Improve tests for verify-package-json
+   - Remove unused component fixture
+   - Verify package.json manifest has description and keywords
+   - Make `obt test --debug` watch for changes and rebuild tests.
+   - expect origamiVersion to be the "2.0" string
+
+- [Financial-Times/o-tracking](https://api.github.com/repos/Financial-Times/o-tracking)
+   - Duplicate click events from multiple browser contexts
+   - Fix bug where loading spoor-id from the cookie would silently fail
+
+- [Financial-Times/origami-repo-data](https://api.github.com/repos/Financial-Times/origami-repo-data)
+   - Only read `keywords` property from the package.json manifest and not origami.json or bower.json
+   - Allow origami version to be the "2.0" string
+   - spec v2 component `Version`s have no `languages`, when they do
+   - Correct the languages property of spec v2 component versions
+   - Move `Version.createFromIngestion` logic.
+   - use v3 of the build service for spec v2 components
+
+- [Financial-Times/o-fonts](https://api.github.com/repos/Financial-Times/o-fonts)
+   - Add FinancierDisplayWeb Light/Semibold support.
+
+- [Financial-Times/create-origami-component](https://api.github.com/repos/Financial-Times/create-origami-component)
+   - Move description and keywords to the package.json
+   - Update v2 workflows to current impl
+
+- [Financial-Times/origami-image-service](https://api.github.com/repos/Financial-Times/origami-image-service)
+   - mention the ECONNREFUSED error so it appears when grepping
+
+- [Financial-Times/polyfill-library](https://api.github.com/repos/Financial-Times/polyfill-library)
+   - Update Intl.DateTimeFormat browser config to match MDN
+   - Element.prototype.getAttributeNames
+
+- [Financial-Times/o-table](https://api.github.com/repos/Financial-Times/o-table)
+   - Allow for multiple filters to be declared declaratively
```

Comment:
FYI the script includes issues and PRs and needs some manual filtering. 

> Allow for multiple filters to be declared declaratively
For example this was an issue that was closed as a duplicate, so we should remove from the email because it sounds like we've added a feature that doesn't exist 🙈